### DISCUSSION TITLE
Feature request or Issue: Device no.

### DISCUSSION THREAD

**Discussion by ArcticApp** [bastianh/streamdeck-vjoy-w4rl0ck#6] [Ideas]
I'm glad you created this app. Its very useful specially when using with the dials on the streamdeck+.  In my set up I have 2 Vjoy devices with 128 buttons each. The current issue is that when configuring the buttons the same device is assigned to all the buttons globally.  Changing the device no on a new button changes the device no on all existing buttons. Can you please allow different vjoy device for each button.  Keep up the good work.  Thank you.

**Comment by bastianh** [comment]
Unfortunatly it's not possible with how the plugin works without rewriting a huge part of it... it was a design decision in the beginning that I preferred to be able switch the device easily and e.g. import/export profiles to share them with others, so if they use different devices e.g. for joystick gremlin they don't need to change every button one by one.
I was pretty sure that 128 buttons would be enough... and with using the pov buttons you could add more, but of course those have limitations.
